### PR TITLE
added alter script file

### PR DESCRIPTION
**Pull request by ryanknauer**
#33

### CODE REVIEW

**Comment by shreyansjain** [comment]
I think, wait for PR #31 to be merged and then `git pull --rebase origin master` and then push it. it'll remove one of the files.
I am expecting you might have to force-push

**Comment by shreyansjain** [comment]
Wait for #31 to be merged first

File: api-constants/functions/GetRunnerUpDiscounts/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        try {
+            const query = SQL `
+                                SELECT c.id, c.description
+                                FROM   code_runner_up_discount c
+                            `;
+            let result = yield client.query(query);
+            if (result.rowCount) {
+                let runner_up_discounts = result.rows
+                let runner_up_discountsResult = runner_up_discounts.map(function(gender) {
+                    return new Code_Runner_Up_Discount(gender.id, gender.description).toJSON();
```
Comment (anmolg) [inline api-constants/functions/GetRunnerUpDiscounts/index.js]: Why are we naming the attribute gender?

File: api-constants/functions/GetRunnerUpDiscounts/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    console.log(event);
+
+    context.callbackWaitsForEmptyEventLoop = false;
+
+    Promise.coroutine(function* () {
+
+        var client = yield pool.connect();
+        try {
+            const query = SQL `
+                                SELECT c.id, c.description
```
Comment (anmolg) [inline api-constants/functions/GetRunnerUpDiscounts/index.js]: Indent level 1

File: api-constants/functions/GetRunnerUpDiscounts/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                                FROM   code_runner_up_discount c
+                            `;
+            let result = yield client.query(query);
+            if (result.rowCount) {
+                let runner_up_discounts = result.rows
```
Comment (anmolg) [inline api-constants/functions/GetRunnerUpDiscounts/index.js]: Confirm rowCount > 0

**Pull request merged**